src/libDirectoryService/DirectoryService.cpp

```diff
@@ -90,7 +97,7 @@ void DirectoryService::StartSynchronization(bool clean) {
   };
 
   auto func2 = [this]() -> void {
-    if (!m_mediator.m_lookup->GetDSInfoLoop()) {
+    while (!m_mediator.m_lookup->GetDSInfoLoop()) {
       LOG_GENERAL(WARNING, "Unable to fetch DS info");
```

Comment:
Should we need to sleep here before trying again? To avoid some kind of busy loop

Reply:
`GetDSInfoLoop` itself wait and retries multiple time. So we don't need additional sleep time over here.

Reply:
ok